mysql-test/suite/rocksdb/t/rocksdb.test

```diff
@@ -825,6 +829,7 @@ send update t49 set a = 1000 where pk = 1;
 --connect (con2,localhost,root,,)
 --echo kill query \$con1_id;
 --disable_query_log
+# this is bad and will result in unstable tests
```

Comment:
I thought this in itself was a marker, along with new MYR issue that this test needs to be destroyed and started over.